routes/accounts.js

```diff
@@ -160,152 +319,55 @@ module.exports = torIps => {
 				});
 			});
 		} else if (accountCreationDisabled.status && !hasBypass) {
-			res.status(403).json({
-				message:
-					'Creating new accounts is temporarily disabled most likely due to a spam/bot/griefing attack.  If you need an exception, please contact our moderators on discord.'
+			const creationDisabledSignup = new Signups({
+				date: new Date(),
+				userName: username,
+				type: 'Failed - ACD',
+				ip: obfIP(signupIP),
+				email: Boolean(email),
+				unobfuscatedIP: signupIP
+			});
+			creationDisabledSignup.save(() => {
+				res.status(403).json({
+					message:
+						'Creating new accounts is temporarily disabled most likely due to a spam/bot/griefing attack.  If you need an exception, please contact our moderators on discord.'
+				});
 			});
 		} else if (torIps.includes(signupIP)) {

```

Comment:
If `getipintel` is reliable we could actually rip out all of the tor stuff I did in the last week, doesn't matter if they're using tor or a vpn.

Reply:
I've found getipintel to be pretty reliable but I think it might be better to have the distinction simply because I think there might be genuine users trying to access the site via VPN, but I don't really see us needing to accommodate those on TOR ever, so TOR doesn't get bypassed with a key, but the VPN restriction can be.

Reply:
Sounds good 👍